61_network_comparison.Rmd

```diff
@@ -0,0 +1,323 @@
+
+Performing **differential network analysis** is challenging because network measures do not follow classical statistical distributions. @shojaie2021differential provide an overview of differential network analysis methods, but focus only on changes in edge sets. @lichtblau2017comparative compare differential network analysis methods that incorporate multiple local and global network measures. @jardim2019bionetstat present a tool "BioNetStat" for differential analysis of biological networks, which is able to compare certain network measures between groups. 
+
+Here we use [NetCoMi](https://github.com/stefpeschel/NetCoMi) [@peschel2021netcomi] for network comparison, which includes the following **differential network analysis approaches:**
+
+- **Permutation approach** to test global network measures (e.g., transitivity, connectivity, or average path length) as well as centrality measures for group differences.
+- **Jaccard** index to assess the similarity between sets of most central nodes
+- **Adjusted Rand index** to assess the similarity between clusterings
+- **Graphlet Correlation Distance** (GCM)
+
+See [@peschel2021netcomi] for an explanation of the first three approaches. The GCM was proposed by @yaverouglu2014revealing.
+
+Two methods (Fisher's z-test [@fisher1970statistical] and the Discordant method [@siska2016discordant]) are available for identifying differential correlations, and **permutation tests** for the more general case of identifying **differential associations**. See [@peschel2021netcomi] for details. NetCoMi offers also a function for plotting a differential network. How to install NetCoMi from GitHub is explained in chapter \@ref(network-learning). 
+
+The **PeerJ data set** [@potbhare2022skin] containing skin microbial profiles for 58 subjects is again used in this chapter. The dataset also includes information on the subjects' geographic location, gender, age and diet. Whether the **skin microbiome differs** between people with **different diets** is an interesting question that will be explored in this chapter.
+
+## Data preparation
+
+We perform the same data preprocessing steps as in chapter \@ref(network-learning).
+
+```{r load_packages, message=FALSE, warning=FALSE}
+library(NetCoMi)
+library(mia)
+```
+
+```{r load_data}
+data("peerj13075", package = "mia")
+tse <- peerj13075
+```
+
+```{r preprocessing}
+# Agglomerate by genus and subset by prevalence
+tse <- subsetByPrevalentFeatures(tse,
+                                 rank = "genus",
+                                 prevalence = 0.2,
+                                 detection = 0)
+
```

Comment:
Note comments from the earlier chapter. This is not agglomerating, this is filtering.